lesson_16/homework_16.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-class-docstring
+# pylint: disable=missing-function-docstring
+
+
+"""Завдання 1
+Створіть клас Employee, який має атрибути name та salary. Далі створіть два класи, Manager та Developer,
+які успадковуються від Employee. Клас Manager повинен мати додатковий атрибут department, а клас Developer -
+атрибут programming_language.
+Тепер створіть клас TeamLead, який успадковується як від Manager, так і від Developer. Цей клас представляє
+керівника з
+команди розробників. Клас TeamLead повинен мати всі атрибути як Manager (ім''я, зарплата, відділ),
+а також атрибут
+team_size, який вказує на кількість розробників у команді, якою керує керівник.
+Напишіть тест, який перевіряє наявність атрибутів з Manager та Developer у класі TeamLead"""
+
+
+class Employee:
+    def __init__(self, name, salary, **kwargs):
+        self.name = name
+        self.salary = salary
+
+class Manager(Employee):
+    def __init__(self, name, salary, department):
+        self.department = department
+        Employee.__init__(self, name, salary)
```

Comment:
please rewrite all to super() construction. Better to use super cause this remove hardcodes ( name of classes)